Sources/Plasma/FeatureLib/pfPython/pyImage.cpp

```diff
@@ -189,10 +189,10 @@ void pyImage::SaveAsJPEG(const plFileName& fileName, uint8_t quality)
     plJPEG::Instance().WriteToFile(fileName, this->GetImage());
 }
 
-void pyImage::SaveAsPNG(const plFileName& fileName)
+void pyImage::SaveAsPNG(const plFileName& fileName, std::multimap<plString, plString>* textFields)
```

Comment:
Since the code this eventually gets passed to checks for `nullptr` and then defaults it with:
```c++
auto textFields = new std::multimap<plString, plString>();
```
It seems reasonable to just make this a const reference and set the default to an empty multimap (rather than `nullptr`)